src/config/config_base.py

```
@@ -43,7 +43,7 @@ def _convert_field(cls, value: Any, field_type: Any) -> Any:
             if origin is tuple:
                 if len(value) != len(args):
                     raise TypeError("Tuple length mismatch")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)